### ISSUE TITLE
Backend server start failed

### ISSUE THREAD

**Issue by AtticusHuang** [derekhe/msfs2020-map-enhancement#297]
我又来了，现在升7.0之后不管啥地图都会backend server start failed，但6.2.3在同样环境下啥毛病没有
同时，License页面里点了copy device id之后没复制出来啥东西，但软件里写着copied，估计是没生成出来
还有就是我才发现你原来是国人，整的我前天写洋文都快怀疑人生了 :)
**MSFS version:**
 - msstore
 - MSFS2020 Version [1.29.30.0]

**Mod version:**
- 7.0.0

**Device ID**
- 没复制出来

**Log**
- 没找着在哪:(

**Screenshots**

![Snipaste_2022-12-04_17-23-15](https://user-images.githubusercontent.com/97792909/205483385-6f412608-ea19-401b-8c99-cf52889a7d36.png)
![Snipaste_2022-12-04_17-15-28](https://user-images.githubusercontent.com/97792909/205483387-57e3635f-c414-42b0-b694-8149114f1824.png)
![Snipaste_2022-12-04_17-14-59](https://user-images.githubusercontent.com/97792909/205483388-8be12bf0-9b4e-485c-855f-a834ab07a67d.png)

**Additional context**

**Comment by AtticusHuang** [comment]
还有可能是Image server test failed，但6.2.3在一样的环境下仍然没毛病
![Snipaste_2022-12-04_17-29-50](https://user-images.githubusercontent.com/97792909/205483560-c5db3165-9ae5-4a70-b132-8290aa11c08c.png)

**Comment by derekhe** [comment]
是的哈，我正在修这个问题

**Comment by wmstapp** [comment]
I'm having exact same issue.
![err](https://user-images.githubusercontent.com/45436083/205504509-cce5458a-2a2b-4ba1-bc76-cd3c196e774a.PNG)

**Comment by derekhe** [comment]
Please give me the log under the installation folder\resources\core\release\log
There are a couple of .log files. Please zip these files and attach it to this issue.

**Comment by AtticusHuang** [comment]
[log.zip](https://github.com/derekhe/msfs2020-map-enhancement/files/10153853/log.zip)

**Comment by AtticusHuang** [comment]
哦7.0.3好像已经没这问题了。

**Comment by derekhe** [comment]
good

**Comment by wmstapp** [comment]
Hello,

I already resolved the issue.  I found that there was still a version of
the 6x installed. I went to Add/remove programs and it listed both the 6x
and 7.0.2 versions. I uninstalled both and downloaded the fresh install
files for 7.03 from the site and installed and it is now working.  I had
originally updated the app using the built in updater... apparently, it
didn't fully update or remove the 6x version.

Hope this helps!

Cheers!
-Mike
